src/core/server/plugins/discovery/plugin_manifest_parser.ts

```diff
@@ -67,7 +67,10 @@ const KNOWN_MANIFEST_FIELDS = (() => {
  * @param packageInfo Kibana package info.
  * @internal
  */
-export async function parseManifest(pluginPath: string, packageInfo: PackageInfo) {
+export async function parseManifest(
+  pluginPath: string,
+  packageInfo: PackageInfo
+): Promise<PluginManifest> {
```

Comment:
__question:__ what is the reason for this change in this PR?

Reply:
to be explicit, but can revert, it doesn't bring lots of value now